mlir/tools/mlir-opt/mlir-opt.cpp

```diff
@@ -331,6 +334,7 @@ int main(int argc, char **argv) {
   ::test::registerTestTransformsTransformDialectExtension(registry);
   ::test::registerTestTilingInterfaceTransformDialectExtension(registry);
   ::test::registerTestDynDialect(registry);
+  ::test::registerIrdlTestDialect(registry);
```

Comment:
While you're at it could you sort this alphabetically too if that doesn't break anything (just try it and if CI complain let's reverse)?